### ISSUE TITLE
Release Error

### ISSUE THREAD

**Issue by codsec** [luandersonn/MyFTP#41]
A aplicação funciona normalmente quando eu rodo ela atraves do VS mas quando eu compilo e tento rodar o MyFTP.exe nada acontece, simplesmente nada funciona...
(o app nem abre)

**Comment by luandersonn** [comment]
Fora do VS?

**Comment by codsec** [comment]
> Fora do VS?

Sim, quando eu compilo ele no visual studio e abro a pasta release  executo o mesmo, nada acontece... enquanto eu rodo ele atraves do debug/release atraves do vs etc funciona normalmente, resumidamente não consigo ter a versão compilada do programa para poder instalar nas maquinas e trabalhar com ele...

**Comment by codsec** [comment]
![image](https://user-images.githubusercontent.com/44205014/211080709-030abcd8-3eba-4bdd-a5d1-a38407b25cd3.png)

**Comment by luandersonn** [comment]
E quando você tenta iniciar pelo ícone no menu iniciar?
